src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1193,7 +1193,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
     }
 
     // Method Description:
-    // - Actually handle a scrolling event, with from a mouse wheel or a
+    // - Actually handle a scrolling event, wether from a mouse wheel or a

```

Comment:
```suggestion
    // - Actually handle a scrolling event, whether from a mouse wheel or a

```

And I don't know how spellcheck bot didn't catch this. Should figure that out too.

Reply:
Oh boy this and a VTE thing I just automerged over in #5200, that's weird. Well `master` is busted now, so I'll go fix that up.